.github/workflows/publish.yml

```diff
@@ -16,6 +16,25 @@ jobs:
           w3up-proof: ${{ secrets.W3UP_PROOF }}
           w3name-private-key: ${{ secrets.W3NAME_PRIVATE_KEY }}
           w3name-revision: ${{ secrets.W3NAME_REVISION }}
+      - if: failure()
+        uses: slackapi/slack-github-action@v2.0.0
+        with:
+          method: chat.postMessage
+          token: ${{ secrets.SLACK_BOT_TOKEN }}
+          payload: |
+            {
+              "channel": "alerts",
```

Comment:
What do you think about adding separate channel for CI alerts? Since we're only reporting publish failure maybe it does makes more sense to keep it under alerts, but in case we're going to report test or lint failures we might get alert fatigue.

Reply:
Agreed, for other reports we should add a separate channel, in order not to fatigue the alert person. But, for now, this is only deployment and publish failure, which is proper alert stuff